src/osd/OSD.cc

```diff
@@ -6124,9 +6124,9 @@ void OSD::do_command(Connection *con, ceph_tid_t tid, vector<string>& cmd, buffe
       f->dump_int("blocksize", bsize);
       f->dump_unsigned("bytes_per_sec", rate);
```

Comment:
JSONs allow floating point (i.e. `f->dump_float()`, why stick to integer )?

Reply:
This patch is just about stdout vs. stderr, feel free to open a separate PR for other changes

Reply:
Yes, You are right, I will.

But, I found the same bug about stdou/stderr for "debug dump_missing" and "getomap" commands

Reply:
https://github.com/ceph/ceph/pull/21960